.gitignore

```diff
@@ -6,12 +6,16 @@
 
 .idea/misc.xml
 
+.idea/preferred-vcs.xml
```

Comment:
Here's the official JetBrains documentation on what to share:
https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

Let's follow their recommendations, starting with WebWorldWind. Based on their recommendations we need to commit preferred-vcs.xml. That said, please be sure that the Windows laptop is running a current WebStorm version, and that the project is associated with GitHub (i.e. you're not using a command line client on the side).